### PR TITLE
add mobile broadband mbbcx client driver sample code

### PR DESCRIPTION
**Pull request by taj0207**
add the first mbbcx client driver 'cxwmbclass' sample code built with vs 2022.

### CODE REVIEW

File: network/wwan/cxwmbclass/BusFastIO.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+#include <precomp.h>
+
+VOID MbbBusSendNetBufferLists(_In_ MBB_BUS_HANDLE BusHandle, _In_ PNET_BUFFER_LIST NetBufferList, _In_ ULONG SessionId, _In_ ULONG SendFlags)
+{
+    PBUS_OBJECT BusObject = (PBUS_OBJECT)BusHandle;
+    BusObject->SendNetBufferListsHandler(BusObject->ModemContext, NetBufferList, SessionId, SendFlags);
+}
+
+VOID MbbBusReturnNetBufferLists(_In_ MBB_BUS_HANDLE BusHandle, _In_ PNET_BUFFER_LIST NetBufferList, _In_ ULONG ReturnFlags)
+{
```
Comment (taj0207) [inline network/wwan/cxwmbclass/BusFastIO.cpp:14]: t

File: network/wwan/cxwmbclass/BusFastIO.cpp [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+//    Copyright (C) Microsoft.  All rights reserved.
+//
+
+#include <precomp.h>
+
+VOID MbbBusSendNetBufferLists(_In_ MBB_BUS_HANDLE BusHandle, _In_ PNET_BUFFER_LIST NetBufferList, _In_ ULONG SessionId, _In_ ULONG SendFlags)
+{
+    PBUS_OBJECT BusObject = (PBUS_OBJECT)BusHandle;
+    BusObject->SendNetBufferListsHandler(BusObject->ModemContext, NetBufferList, SessionId, SendFlags);
+}
```
Comment (taj0207) [inline network/wwan/cxwmbclass/BusFastIO.cpp:11]: test

File: network/wwan/cxwmbclass/BusFastIO.cpp [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+VOID MbbBusReturnNetBufferLists(_In_ MBB_BUS_HANDLE BusHandle, _In_ PNET_BUFFER_LIST NetBufferList, _In_ ULONG ReturnFlags)
+{
+    PBUS_OBJECT BusObject = (PBUS_OBJECT)BusHandle;
+    BusObject->ReturnNetBufferListsHandler(BusObject->ModemContext, NetBufferList, ReturnFlags);
```
Comment (taj0207) [inline network/wwan/cxwmbclass/BusFastIO.cpp:16]: test2

File: network/wwan/cxwmbclass/BusFastIO.cpp [hunk 4]
```diff
@@ -0,0 +1,85 @@
+//
+//    Copyright (C) Microsoft.  All rights reserved.
+//
+
+#include <precomp.h>
+
+VOID MbbBusSendNetBufferLists(_In_ MBB_BUS_HANDLE BusHandle, _In_ PNET_BUFFER_LIST NetBufferList, _In_ ULONG SessionId, _In_ ULONG SendFlags)
+{
```
Comment (taj0207) [inline network/wwan/cxwmbclass/BusFastIO.cpp:8]: test3

File: network/wwan/cxwmbclass/BusFastIO.cpp (hunk 2 above)
Comment (taj0207) [inline network/wwan/cxwmbclass/BusFastIO.cpp:11]: test4

File: network/wwan/cxwmbclass/BusFastIO.cpp [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    PBUS_OBJECT BusObject = (PBUS_OBJECT)BusHandle;
+    BusObject->SendNetBufferListsHandler(BusObject->ModemContext, NetBufferList, SessionId, SendFlags);
+}
+
+VOID MbbBusReturnNetBufferLists(_In_ MBB_BUS_HANDLE BusHandle, _In_ PNET_BUFFER_LIST NetBufferList, _In_ ULONG ReturnFlags)
+{
+    PBUS_OBJECT BusObject = (PBUS_OBJECT)BusHandle;
```
Comment (taj0207) [inline network/wwan/cxwmbclass/BusFastIO.cpp:15]: test5

File: network/wwan/cxwmbclass/BusFastIO.cpp [hunk 6]
```diff
@@ -0,0 +1,85 @@
+//
+//    Copyright (C) Microsoft.  All rights reserved.
```
[the remark this answers is not in the document]
Comment (taj0207) [reply network/wwan/cxwmbclass/BusFastIO.cpp]: removed

File: network/wwan/cxwmbclass/inc/BusFastIO.h [hunk 7]
```diff
@@ -0,0 +1,35 @@
+#pragma once
```
[the remark this answers is not in the document]
Comment (taj0207) [reply network/wwan/cxwmbclass/inc/BusFastIO.h]: removed

File: network/wwan/cxwmbclass/inc/mbbfastio.h [hunk 8]
```diff
@@ -0,0 +1,102 @@
+//This header file depends on <ndis.h> and defines NDIS_SUPPORT_NDIS670 or higher
```
[the remark this answers is not in the document]
Comment (taj0207) [reply network/wwan/cxwmbclass/inc/mbbfastio.h]: removed

File: network/wwan/cxwmbclass/selsuspend.cpp [hunk 9]
```diff
@@ -0,0 +1,79 @@
+//
```
[the remark this answers is not in the document]
Comment (taj0207) [reply network/wwan/cxwmbclass/selsuspend.cpp]: removed

**Review by tianye606 — APPROVED** [review]

**Pull request merged**